### PR TITLE
Update events css

### PR DESCRIPTION
**Pull request by apanjain**

### CODE REVIEW

**Comment by apanjain** [comment]
some of the css of events mobile view has been edited in pr#36

File: frontend/src/components/EventCard/EventCard.module.scss [hunk 1]
```diff
@@ -54,40 +43,31 @@
...
+@mixin heading-mixin{
     text-align:left;
     display:block;
     font-weight: bold;
-    color: rgba(0, 0, 0, 0.8);
+    color: $grey-level-0_8;
     margin: 0.625rem auto 0.625rem 0;
+}
+.headingsm {
+    @include heading-mixin;
```
Comment (hacksdump) [inline frontend/src/components/EventCard/EventCard.module.scss:54]: On a second note, inheritance would be more suitable in this scenario. Create a class called heading and let other heading classes inherit from it as `@extend .heading`

File: frontend/src/components/EventCard/EventCard.module.scss [hunk 2]
```diff
@@ -1,32 +1,20 @@
...
 
 .containerlg{
-    font-size: 36.9375rem;
-    widows: 80%;
     max-width: 90vw;
-    // height:fit-content;
 
 }
 
 .containerxl{
```
Comment (hacksdump) [inline frontend/src/components/EventCard/EventCard.module.scss:16]: The content is still not centre aligned like the design.
Screenshot from 2020-04-09 02-16-40

File: frontend/src/components/EventCard/EventCard.module.scss (hunk 1 above)
Comment (hacksdump) [reply frontend/src/components/EventCard/EventCard.module.scss:54]: Keep the mixin, use it in parent class and inherit the parent class in child classes.

**Pull request merged**